### ISSUE TITLE
Parse error with 'extract' function (Postgresql)

### ISSUE THREAD

**Issue by olilau** [tobymao/sqlglot#7]
Hello,

This code:
```
import sqlglot
sqlglot.transpile("select extract(month from '2021-01-31'::timestamp without time zone)", read='postgres', pretty=True)
```
raises an error:
ParseError: Expected ). Line 0, Col: 26.
select LPAD(extract(month from '2021-01-31'::timestamp without time zone)::text,

The query works without any issue in Postgresql.

Here is the complete traceback:
[tb_sqlglot.txt](https://github.com/tobymao/sqlglot/files/6883277/tb_sqlglot.txt)

Thanks.

Best,

**Comment by tobymao** [comment]
released in v1.2.0 thanks for reporting this

**Comment by olilau** [comment]
You fix works like a charm. Thanks.
